centos_ci.sh

```diff
@@ -293,26 +293,32 @@ function perform_release() {
 }
 
 function perform_docs_publish() {
-  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
-  REPO_OWNER=$(echo $REPO | cut -d"/" -f4)
   cd gopath/src/github.com/minishift/minishift
+  REPO="https://github.com/$1/minishift"
 
-  git checkout master
-  git checkout -b docs-branch
-  git pull -f $REPO $BRANCH # Get changes from remote repo/branch
-  git log -n 1              # Display last commit in log as reference
+  git remote add remote-repo $REPO
+  git fetch remote-repo
+  git checkout -b docs-branch --track remote-repo/$BRANCH
+  git log -n 1 # Display last commit in log as reference
 
   install_docs_prerequisite_packages;
   make gen_adoc_tar
   build_openshift_origin_docs $(pwd)/docs/build/minishift-adoc.tar;
 
-  mkdir -p minishift/docs/ondemand/$REPO_OWNER-$BRANCH
-  cp -r openshift-docs/_preview/openshift-origin minishift/docs/ondemand/$REPO_OWNER-$BRANCH/ # Copy the openshift-origin
-  cp docs/build/minishift-adoc.tar minishift/docs/ondemand/$REPO_OWNER-$BRANCH/ # Copy doc tar
-  # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working
-  RSYNC_PASSWORD=$RSYNC_PASSWORD rsync -av --delete --relative minishift/docs/ondemand/$REPO_OWNER-$BRANCH/ minishift@artifacts.ci.centos.org::minishift/
-  echo "Find docs tar here http://artifacts.ci.centos.org/minishift/minishift/docs/ondemand/$REPO_OWNER-$BRANCH/"
-  echo "Minishift documentation is hosted at http://artifacts.ci.centos.org/minishift/minishift/docs/ondemand/$REPO_OWNER-$BRANCH/openshift-origin/latest/minishift/index.html"
+  # Publish documentation for master build
+  if [[ "$JOB_NAME" = "minishift" ]]; then
+    mkdir -p minishift/docs/master
+    cp -r openshift-docs/_preview/openshift-origin minishift/docs/master/ # Copy the openshift-origin
+    # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working
+    RSYNC_PASSWORD=$RSYNC_PASSWORD rsync -av --delete --relative minishift/docs/master/ minishift@artifacts.ci.centos.org::minishift/
+    echo "Minishift documentation is hosted at http://artifacts.ci.centos.org/minishift/minishift/docs/master/openshift-origin/latest/minishift/index.html"
```

Comment:
I would suggest `Documentation from Minishift master branch compiled and hosted at LOCATION` . Mention of Openshift Origin does not add any value IMO

Reply:
@robin-owen , Let me know your opinion on above comment. Does it make sense to remove OpenShift reference here at all to avoid any ambiguity or does the updated text sounds fine to you.

Reply:
@LalatenduMohanty @budhrg Personally, I see value in mentioning that it's integrated with the documentation for OpenShift. In this case, it's a matter of expectation, so knowing what I'm about to view is valuable. The updated text avoids the ambiguity in my opinion.